### PR TITLE
[docs]: use terminal blocks instead of code blocks for terminal commands

### PR DESCRIPTION
**Pull request by kbrandwijk**
# Why

# How

# Test Plan

# Checklist

- [X] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] This diff will work correctly for `expo build` (eg: updated `@expo/xdl`).
- [ ] This diff will work correctly for `expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/distribution/release-channels.md [hunk 1]
```diff
@@ -102,12 +102,7 @@ Environment variables don't exist explicitly, but you can utilize release channe
 
 Say you have a workflow of releasing builds like this:
 
-- `expo publish --release-channel prod-v1`
-- `expo publish --release-channel prod-v2`
-- `expo publish --release-channel prod-v3`
-
-- `expo publish --release-channel staging-v1`
-- `expo publish --release-channel staging-v2`
+<TerminalBlock cmd={['# Publish to release channel prod-v1', 'expo publish --release-channel prod-v2', '', '# Publish to release channel prod-v2', 'expo publish --release-channel prod-v1', '', '# Publish to release channel prod-v3', 'expo publish --release-channel prod-v3', '', '', '# Publish to release channel staging-v1', 'expo publish --release-channel staging-v1', '', '# Publish to release channel staging-v2', 'expo publish --release-channel staging-v2', '']} />
```
Comment (Simek) [inline docs/pages/distribution/release-channels.md:105]:
```suggestion
<TerminalBlock cmd={['# Publish to release channel prod-v1', 'expo publish --release-channel prod-v2', '', '# Publish to release channel prod-v2', 'expo publish --release-channel prod-v1', '', '# Publish to release channel prod-v3', 'expo publish --release-channel prod-v3', '', '', '# Publish to release channel staging-v1', 'expo publish --release-channel staging-v1', '', '# Publish to release channel staging-v2', 'expo publish --release-channel staging-v2']} />  
```

This results in unnecessary empty line ad the end of block.

**Review by Simek — CHANGES_REQUESTED** [review]
Hi @kbrandwijk, 

it looks like you have missed the import for the `TerminalBlock` component at the top of file:
```js
import TerminalBlock from '~/components/plugins/TerminalBlock';
```

Besides that, I have left one formatting suggestion.

File: docs/pages/distribution/release-channels.md (hunk 1 above)
Comment (kbrandwijk) [reply docs/pages/distribution/release-channels.md:105]: Good catch!

**Review by Simek — APPROVED** [review]
Thank you for the updates! 🙏

**Pull request merged**